npu.user.js

```diff
@@ -506,8 +507,11 @@ var npu = {
 					var select = $('<ul class="termSelect"></ul>');
 					var stored = npu.getUserData(null, null, ["termSelect", npu.getPage()]);
 					var found = false;
+					var admissionSemester = $("#lblTrainingName").text().split("[")[0].split(":")[1];

```

Comment:
This seems a bit fragile to me. What if the training name contains a "[" character, followed by a letter? Then we would be removing all semesters from the term selector as numbers come before letters.

The "subject name parsing" bugs I had to fight a while ago taught me never to underestimate the amount of weird or badly formatted data out there. What seems consistent here is that the end of the string is always in the form of `:XXXX/YY/Z[some text]`. Maybe a regex for that?